includes/widgets/table.php

```diff
@@ -188,7 +189,66 @@ protected function register_controls() {
 			]
 			);
 
+		
+		$this->end_controls_section();
+
+		$this->start_controls_section(
+			'style_section',
+			[
+				'label' => esc_html__( 'Style', 'elementor' ),
+				'tab' => \Elementor\Controls_Manager::TAB_STYLE,
+			]
+		);
+
+		$this->add_control(
+			'table_head_color',
+			[
+				'label' => esc_html__( 'Table head Color', 'elementor' ),
```

Comment:
Use labels from Trello.

Using the correct lowercase vs uppercase is important. You can't mix it randomly.